Documentation/CraftingNuGetPackages.md

```diff
@@ -0,0 +1,69 @@
+# Crafting a NuGet Package for MSB4U
+
+> MSBuildForUnity consumes .NET based packages without any modifications to the package. This documentation describes how C++, WinRT or packages with Unity-specific contents can easily be adapted to work with MSBuildForUnity.
+
+## Quickstart Summary
+
+Though the use of a [Build Targets or Props](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#include-msbuild-props-and-targets-in-a-package) file inside a NuGet package, we can leverage properties provided by MSBuildForUnity to appropriately select the contents to bring into the Unity project. The following snippet is taken from [Microsoft.Windows.MixedReality.DotNetWinRT](https://www.nuget.org/packages/Microsoft.Windows.MixedReality.DotNetWinRT/) NuGet package. It brings in the `Unity` folder packaged in the root of the NuGet package to the output folder inside Unity.
```

Comment:
nit: we can leverage properties provided by MSBuildForUnity to appropriately select the contents to bring into the Unity project.

Right now its unclear which project we are selecting content for. I think that maybe this is to specify how content in your nuget package gets added to the package consumer's unity project?

we can leverage properties provided by MSBuildForUnity to appropriately select the contents to bring into the package consumer's Unity project.